src/main/java/com/exadel/discountwebapp/discount/controller/DiscountController.java

```diff
@@ -43,6 +45,18 @@ public DiscountResponseVO update(@PathVariable Long id, @Valid @RequestBody Disc
         return discountService.update(id, request);
     }
 
+    @PreAuthorize("hasAuthority('ADMIN')")
+    @PostMapping("/{id}/tags")
+    public List<TagResponseVO> addTags(@PathVariable Long id, @Valid @RequestBody List<Long> tagIds) {
```

Comment:
let's link discount with tags during the discount creation (list of tag ids should be in the discount request)

Reply:
done